app/workers/login_adoption_email_job.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class LoginAdoptionEmailJob
+  include Sidekiq::Worker
```

Comment:
Is there a reason you're using sidekiq here? It appears that you are using this synchronously. Should it be async? If this is going to be used synchronously it would make more sense for it to be a service instead of a worker.